.github/workflows/test.yml

```diff
@@ -28,10 +28,34 @@ jobs:
           pip install pylint
           if [ -f setup.py ]; then pip install -e .; fi
           pip install transformers
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          bash Miniconda3-latest-Linux-x86_64.sh -b -u
       - name: Lint with PyLint
         run: |
           pylint src --rcfile .pylintrc
       - name: Test with unittest
         run: |
           python test/cli.py
-          python test/analysis.py
+          python test/analysis.py    
+      - name: Install and Start Slurm
+        run: |
+          sudo apt update -y
+          sudo apt install slurm-wlm -y
+          cp test/helpers/slurm.conf test/helpers/slurm_modified.conf
+          sed -i "s/YOUR_HOSTNAME_HERE/$HOSTNAME/" test/helpers/slurm_modified.conf
+          sudo mv test/helpers/slurm_modified.conf /etc/slurm/slurm.conf
+          sudo service slurmd start
+          sudo service slurmctld start
+          sudo service munge start
+      - name: Test benchit on Slurm
+        run: |
+          # Create conda environment for Slurm using srun (sbatch + wait)
+          source ~/miniconda3/bin/activate
```

Comment:
This gets replaced by the method in #104

Reply:
See approach: https://github.com/groq/mlagility/pull/104/commits/cdaea11bd7a0a9944719aba00ab11855368afd8c